testing/run_tests.py

```diff
@@ -543,7 +546,8 @@ def make_test(name, flags=None, extra_env=None):
         shuffle_flags + [
             '--enable_vulkan_validation',
             # TODO(https://github.com/flutter/flutter/issues/142642): Remove this.
-            '--gtest_filter=-*OpenGLES',
+            # TODO EVEN MORE
+            '--gtest_filter=*Metal',
```

Comment:
The vulkan variants crash, I think this is because they weren't running before? I'd need to double check. Either that or they are just hitting validation checks that are fatal in a way the test harness can't handle